skops/card/_model_card.py

```diff
@@ -241,15 +241,8 @@ def __init__(
         metadata: Optional[CardData] = None,
     ) -> None:
         self.model = model
-        self._hyperparameter_table = self._extract_estimator_config()
-        # the spaces in the pipeline breaks markdown, so we replace them
+        self._model_diagram = model_diagram
```

Comment:
since this is a constructor parameter, it doesn't have to be private.

```suggestion
        self.model_diagram = model_diagram
```